atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,292 @@
+            "text": found_text["properties"]["text"],
+            "competency_ids": new_text_competencyID
+        }]}
+
+        self.weaviate_client.update_property_by_id(CollectionNames.TEXT.value, found_text["properties"]["text_id"], new_text)
+
+        members = self.weaviate_client.get_embeddings_by_property(CollectionNames.TEXT.value, "competency_ids",
+                                                                  competency_id)
+        cluster_vector = np.array(found_cluster["vector"]["default"])
+        text_vector = np.array(found_text["vector"]["default"])
+
+        newMedoid = update_cluster_centroid(cluster_vector, len(members), text_vector)
```

Ruff (0.11.9)

Variable `newMedoid` in function should be lowercase

(N806)